net/isc-dhcp/Makefile

```diff
@@ -9,24 +9,25 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=isc-dhcp
 UPSTREAM_NAME:=dhcp
-PKG_VERSION:=4.4.3-P1
+PKG_REALVERSION:=4.4.3-P1
+PKG_VERSION:=4.4.3_p1
```

Comment:
I would have used a transformation line `${PKG_REALVERSION/-P/_p}` so that in the future it's a one line change...

Reply:
Indeed, that sounds like a good idea